beacon-chain/blockchain/checktags_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build !develop
```

Comment:
What is this?

cc @prestonvanloon

Reply:
I have no idea, probably gazelle?

Reply:
`//go:build` is new in go 1.17, i think?